platform/android/java/lib/patches/com.google.android.vending.expansion.downloader.patch

```diff
@@ -216,9 +216,9 @@ index 12edd97ab..a0e1165cc 100644
  import android.telephony.TelephonyManager;
  import android.util.Log;
  
-+// -- GODOT start --
++// -- BLAZIUM start --
 +import android.annotation.SuppressLint;
```

Comment:
This maybe from a different branch, please validate this is required for rebranding